packages/mobile/src/send/SendAmount.tsx

```diff
@@ -245,7 +247,11 @@ function SendAmount(props: Props) {
 
   const onSend = React.useCallback(() => {
     if (!isDollarBalanceSufficient) {
-      dispatch(showError(ErrorMessages.NSF_TO_SEND))
+      dispatch(
+        showError(ErrorMessages.NSF_TO_SEND, null, {
+          amountNeeded: minimumAmount,
+        })
+      )
```

Comment:
I think we want to display the local amount and its symbol.
Similar to what we do for escrows
https://github.com/celo-org/wallet/blob/3aff7b630d899fec38fdb4fb6e2df0828a5bc053/packages/mobile/src/send/SendAmount.tsx#L200-L206

This makes me think we need a function equivalent of `<CurrencyDisplay />` so we can format money amounts as text only for cases like this.
No need to do this in this PR though ;)